1.9/usage/gpu/index.md

```diff
@@ -0,0 +1,21 @@
+---
+post_title: Enable GPU Acceleration
+nav_title: Enable GPU Acceleration
+feature_maturity: experimental
+menu_order: 110
+---
+
+DC/OS now supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
```

Comment:
Suggest `DC/OS supports allocating...` - the word `now` won't make much sense as new versions are released.